COLLABORATOR_GUIDE.md

```diff
@@ -87,6 +86,18 @@ The TC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
+Pull requests should be landed with Jenkins as documented [here]
+(https://github.com/nodejs/node/wiki/Merging-pull-requests-with-Jenkins). 
+
+#### Landing Pull Requests manually (for emergency only)
+Pull requests should be landed with Jenkins as documented [here]
```

Comment:
How about we restore the links and make the sections with two ## instead of four, but preserve this duplicate disclaimer? Basically I'd like to make sure that if people end-up jumping to this section, that they know that this procedure is deprecated outside of exceptional circumstances.